### PR TITLE
Security Updates: Update husky, moment-timezone, terser, parse-url, semver

### PR DESCRIPTION
**Pull request by CarlosQ96**
Related to https://github.com/Giveth/impact-graph/issues/568

### CODE REVIEW

**Comment by CarlosQ96** [comment]
Ran server locally and with docker and everything seems in order. Cronjobs were running I was getting the same messages I got before the upgrade of bull dependencies.

Seems safe.

File: package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "handlebars": "^4.7.7",
     "honeyswap-sdk": "^1.0.0",
     "html-pdf-node": "^1.0.7",
+    "husky": "^4.3.8",
```
Comment (mohammadranjbarz) [inline package.json]: How about setting exact version for these dependencies?
(for instance `4.3.8` instead of `^4.3.8` to prevent unwanted upgrades

Comment (CarlosQ96) [reply package.json]: You are right. I think there are no more updates actually. 4.3.8 is the last one. 
Thanks

Comment (CarlosQ96) [reply package.json]: Upgrading husky to 5.0.0 will break it FYI. So lets leave it in 4.3.8

Comment (mohammadranjbarz) [reply package.json]: Thanks @CarlosQ96

**Comment by mohammadranjbarz** [comment]
@maryjaf  It was just about updating some packages, if main functionality of stage is ok you can move it to DONE ( they were not risky packages, so even if just staging is up we can move it to DONE without lots of testing)

**Pull request merged**